addons/mail/static/src/scss/composer.scss

```diff
@@ -41,31 +41,48 @@
     z-index: $o-mail-thread-window-zindex + 1;
 
     .o_viewer_content {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
```

Comment:
I don't like diff in this file. All this SCSS file is deprecated, so adding more stuffs is not desirable.
If you need to make changes, please avoid SCSS hierarchy and flatten new styles as much as possible. This drastically eases desire to convert to OWL component.